### PR TITLE
go1.19.4

### PR DESCRIPTION
**Pull request by RTann**
## Description

https://github.com/stackrox/stackrox/pull/4658 started the bump from go1.18.4 to 1.19.4, but it missed `EXPECTED_GO_VERSION` and generating new proto files which were re-styled due to the version bump. This PR hopefully completes the update to go1.19.4.

That original PR was reverted due to test failures. This has been resolved via https://github.com/openshift/release/pull/36193

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed

CI

I updated `EXPECTED_GO_VERSION` and then ran both `make proto-generated-srcs` and `gogen .`

### CODE REVIEW

**Comment by RTann** [comment]
https://github.com/openshift/release/blob/master/ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml#L26 will need to be updated for this to work

**Review by janisz — APPROVED** [review]

File: generated/storage/group.pb.go [hunk 1]
```diff
@@ -98,10 +98,11 @@ func (m *Group) Clone() *Group {
 
 // GroupProperties defines the properties of a group. Groups apply to users when
 // their properties match. For instance:
-//   * If GroupProperties has only an auth_provider_id, then that group applies
```
Comment (rhybrillou) [inline generated/storage/group.pb.go:101]: I suppose the comments in the source proto files should be updated and pushed as well.

File: BUILD_IMAGE_VERSION [hunk 2]
```diff
@@ -1 +1 @@
-stackrox-build-0.3.55
```
Comment (rhybrillou) [inline BUILD_IMAGE_VERSION:1]: Has the issue at the origin of the revert https://github.com/stackrox/stackrox/pull/4768 been understood and resolved ?

Comment (janisz) [reply BUILD_IMAGE_VERSION:1]: `-dirty` was added because `generated/` files were changed

File: generated/storage/group.pb.go (hunk 1 above)
Comment (rhybrillou) [reply generated/storage/group.pb.go:101]: Looking at the `grouped-static-checks` tests, I think the styling changes are required in the proto files to ensure the code re-generation does not erase the fixes later.

Comment (RTann) [reply generated/storage/group.pb.go:101]: sure we can be on the safe side

**Review by rhybrillou — APPROVED** [review]
Looks good.
Please make sure all the proto files at the source of the changed files under `generated` also contain the changes.

**Comment by 0x656b694d** [comment]
Fixes ROX-12582.

**Pull request merged**